hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemE2E.java

```diff
@@ -259,6 +260,9 @@ public void testHttpReadTimeout() throws Exception {
 
   public void testHttpTimeouts(int connectionTimeoutMs, int readTimeoutMs)
       throws Exception {
+    // This is to make sure File System creation goes through before network calls start failing.
+    assumeValidTestConfigPresent(this.getRawConfiguration(), FS_AZURE_ACCOUNT_IS_HNS_ENABLED);
```

Comment:
Why this change is required?, is this failing without this change ?

Reply:
Yes, without this change the test was failing while creating file system itself when someone skips adding this config.
Without this config, FS needs to do getAcl() and getAcl() call was failing with timeout which is what this test is supposed to do. With this config getAcl() call won't be needed and FS creation will be successful. Post that we can assert on he timeout failures.